DQM/L1TMonitor/src/L1TStage2uGT.cc

```diff
@@ -282,7 +391,75 @@ void L1TStage2uGT::analyze(const edm::Event& evt, const edm::EventSetup& evtSetu
            } // end of uGtAlgs
         } // end of BX
      } // selecting LastCollisionInTrain
-  } // end of uGTAlgs = 1
+
+
+   if(algoBitFirstBxInTrain_ != -1 && itr->getAlgoDecisionInitial(algoBitFirstBxInTrain_)) {
+     for(int ibx = uGtAlgs->getFirstBX(); ibx <= uGtAlgs->getLastBX(); ++ibx) {
+       for(auto itr2 = uGtAlgs->begin(ibx); itr2 != uGtAlgs->end(ibx); ++itr2) {
+         for(unsigned int algo=0; algo<prescaledAlgoBitName_.size(); algo++) { 
+           if(itr2->getAlgoDecisionInitial(prescaledAlgoBitName_.at(algo).second)) { 
+             prescaled_first_collision_run_->Fill(ibx, prescaledAlgoBitName_.at(algo).second);
```

Comment:
The filling of the y axis value is wrong. You have to fill the bin number not the algo bit.
`prescaled_first_collision_run_->Fill(ibx, algo+1);`